sp1-prover/src/main.rs

```diff
@@ -0,0 +1,27 @@
+#![no_main]
+
+use aligned_sp1_prover::{calc_auction_hash, decrypt_bidder_data, AuctionData};
+use ecies::SecretKey;
+
+sp1_zkvm::entrypoint!(main);
+
+pub fn main() {
+    let auction_data = sp1_zkvm::io::read::<AuctionData>();
+
+    let pvk = SecretKey::parse_slice(&sp1_zkvm::io::read::<Vec<u8>>())
+        .expect("missing private key to encode bidder data");
+
+    let mut winner_addr = &vec![];
+    let mut winner_amount = 0;
+    for bidder in &auction_data.bidders {
+        let bidder_amount = decrypt_bidder_data(&pvk, bidder);
```

Comment:
Can decrypting only bidder amount improve the prove time?
`let bidder_amount = decrypt_bidder_data(&pvk, bidder.amount)`

Reply:
no, it doesn't